worker/vcs/manager.go

```diff
@@ -59,9 +59,21 @@ func insert(url, insertion string, index int) string {
 func getUrlwithToken(url, subvcs, token string) string {
 	switch subvcs {
 	case api.GITHUB:
+		if token == "" {
+			log.Warn("subvcs is github, but the token is empty, there meybe exist error")
+			break
+		}
 		position := strings.Index(url, "github.com")
+		if position == -1 {
+			log.Warn("subvcs is github and the token is not empty, but the url dosen't contain github.com")
```

Comment:
I think the log could be "Sub VCS is GitHub, but the URL doesn't contain 'github.com'"